mmv1/third_party/terraform/website/docs/r/compute_router_interface.html.markdown

```diff
@@ -45,19 +44,23 @@ or both.
 
 * `vpn_tunnel` - (Optional) The name or resource link to the VPN tunnel this
     interface will be linked to. Changing this forces a new interface to be created. Only
-    one of `vpn_tunnel` and `interconnect_attachment` can be specified.
+    one of `vpn_tunnel`, `interconnect_attachment` or `subnetwork` can be specified.
 
 * `interconnect_attachment` - (Optional) The name or resource link to the
-  VLAN interconnect for this interface. Changing this forces a new interface to
-  be created. Only one of `vpn_tunnel` and `interconnect_attachment` can be
-  specified.
+    VLAN interconnect for this interface. Changing this forces a new interface to
+    be created. Only one of `vpn_tunnel`, `interconnect_attachment` or `subnetwork` can be specified.
 
-* `project` - (Optional) The ID of the project in which this interface's router belongs. If it
-    is not provided, the provider project is used. Changing this forces a new interface to be created.
+* `subnetwork` - (Optional) The URI of the subnetwork resource that this interface
+    belongs to, which must be in the same region as the Cloud Router. When you establish a BGP session to a VM instance using this interface, the VM instance must belong to the same subnetwork as the subnetwork specified here. Changing this forces a new interface to be created. Only one of `vpn_tunnel`, `interconnect_attachment` or `subnetwork` can be specified.
 
-* `region` - (Optional) The region this interface's router sits in. If not specified,
-    the project region will be used. Changing this forces a new interface to be
-    created.
+* `private_ip_address` - (Optional) The regional private internal IP address that is used
```

Comment:
This matches the c.g.c documentation for the field, but that documentation leaves out "If you don't specify the privateIpAddress option, Google tries to find a free IP address from the subnet's primary IP range to assign to the interface." from https://cloud.google.com/network-connectivity/docs/network-connectivity-center/how-to/creating-router-appliances#create-redundant-interfaces. We should consider adding that here, since that's a significant descriptor of the field's behaviour.

Reply:
There's likely a qualifier needed there, such as `If a subnetwork is specified...`?

Reply:
agree, done